po/fr.po

```diff
@@ -2964,26 +2964,26 @@ msgstr "Paramètres du programme"
 msgid "Options"
 msgstr ""
 
-msgid "Defaults and Initial Settings"
-msgstr "Valeurs initiales et paramètres par défaut"
+msgid "Paths"
+msgstr "Chemins"
+
+msgid "Audio"
+msgstr "Audio"
 
 msgid "Reverb"
```

Comment:
I suggest you to change the order of the msgid for reflecting the tab order in the dialog.